app/livechat/client/views/app/livechatCustomFields.js

```diff
@@ -50,5 +56,6 @@ Template.livechatCustomFields.events({
 });
 
 Template.livechatCustomFields.onCreated(function() {
-	this.subscribe('livechat:customFields');
+	this.customFields = new ReactiveVar([]);
+	loadCustomFields(this);
```

Comment:
Pagination is missing here, right?
Otherwise, only the data received from the first fetch will be rendered.
It's the same case we had on other Livechat templates, which were fixed by implementing the `onScroll` event.